src/components/scenes/TransactionListScene.tsx

```diff
@@ -274,16 +271,26 @@ function TransactionListComponent(props: Props) {
   // Change the second dot color and remove the 3rd dot
   const overrideDots: UpdateDots = [undefined, { color: iconColor }, null]
 
+  const backgroundColors = [...theme.assetBackgroundGradientColors]
+  if (iconColor != null) {
+    const scaledColor = multiplyHexColor(iconColor, theme.assetBackgroundColorScale)
+    backgroundColors[0] = scaledColor
```

Comment:
This scaling seems like a change rather than a refactor.

Reply:
Ah, I see, so it's a way to achieve the ability to darken the background color from the theme file (because it's currently set to `#00000000`, transparent black). Now we can darken or lighten the value...wait, we can lighten the value if we pass a scale > 1, so clipping is still an issue that should be handled in the "darkenHexColor" function. I'd argue "scaleHexColor" might be a better name because darkening isn't the only use.